network_diffusion/seeding/drivernode_selector.py

```diff
@@ -0,0 +1,82 @@
+"""A definition of the seed selector based on driver nodes."""
+
+from typing import List
+
+# from network_diffusion.seeding.base_selector import BaseSeedSelector
+from network_diffusion.tpn.tpnetwork import TemporalNetwork
+from network_diffusion.mln.mlnetwork import MultilayerNetwork
+from network_diffusion.mln.actor import MLNetworkActor
+
+from network_diffusion.utils import BOLD_UNDERLINE, THIN_UNDERLINE
+
+from network_diffusion.tpn.functions import driver_nodes
+from network_diffusion.seeding.random_selector import RandomSeedSelector
+from network_diffusion.seeding.betweenness_selector import BetweennessSelector
+from network_diffusion.seeding.degreecentrality_selector import DegreeCentralitySelector
+from network_diffusion.seeding.closeness_selector import ClosenessSelector
+from network_diffusion.seeding.katz_selector import KatzSelector
+from network_diffusion.seeding.kshell_selector import KShellMLNSeedSelector
+from network_diffusion.seeding.mocky_selector import MockyActorSelector
+from network_diffusion.seeding.neighbourhoodsize_selector import NeighbourhoodSizeSelector
+from network_diffusion.seeding.pagerank_selector import PageRankMLNSeedSelector
+from network_diffusion.seeding.voterank_selector import VoteRankSeedSelector
+from network_diffusion.seeding.cbim import CBIMselector
+
+class DriverNodeSelector():
```

Comment:
this class should inherit from `BaseSeedSelector`. Please implement all necessary functions!

Reply:
I have done this.